packages/fast-development-site-react/src/components/site/component-wrapper.tsx

```diff
@@ -97,6 +98,10 @@ class ComponentWrapper extends React.Component<IComponentWrapperProps<IDevSiteDe
         );
     }
 
+    private getThemeBackground(): React.CSSProperties {
+        return this.props.transparentBackground ? null : {background: this.props.background};
+    }
```

Comment:
getThemeBackground is being called regardless of if there is style or not. It's being set regardless of if background is passed as a prop. Suggest a null check here before going forward.

```
if (!this.props.background) { return };
```